hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsSource.java

```diff
@@ -119,7 +146,7 @@ public long getLastTimeStampOfWalGroup(String walGroup) {
    * @param walGroup which group we are getting
    * @return age
    */
-  public long getAgeofLastShippedOp(String walGroup) {
+  public long getAgeOfLastShippedOp(String walGroup) {
```

Comment:
nit: Unfortunately, we can't fix this method naming typo on IA limited private for a maintenance release.

Reply:
Good catch. I have just pushed to fix for that.